File: Service_eHospital_DongNai_A/Implement/Sync_TableName_KeyGenerationService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class Sync_TableName_KeyGenerationService : BaseService<Sync_TableName_KeyGeneration, ISync_TableName_KeyGenerationRepository>
    , ISync_TableName_KeyGenerationService
    {
    private readonly ISync_TableName_KeyGenerationRepository _Sync_TableName_KeyGenerationRepository;
    public Sync_TableName_KeyGenerationService(ISync_TableName_KeyGenerationRepository Sync_TableName_KeyGenerationRepository) : base(Sync_TableName_KeyGenerationRepository)
    {
    _Sync_TableName_KeyGenerationRepository = Sync_TableName_KeyGenerationRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/Sync_TableName_KeyGeneration.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { Sync_TableName_KeyGeneration } from './Sync_TableName_KeyGeneration.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class Sync_TableName_KeyGenerationService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'TableName', 'KeyManage', 'CodeManage'];

    List: Sync_TableName_KeyGeneration[] | undefined;
    ListFilter: Sync_TableName_KeyGeneration[] | undefined;
    FormData!: Sync_TableName_KeyGeneration;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "Sync_TableName_KeyGeneration";
    }
}

